app/controllers/entries_controller.rb

```diff
@@ -3,6 +3,7 @@ class EntriesController < ApplicationController
   def index
     @entries = Entry.all
     @new_entry = Entry.new
+    @new_entry.locations.build
```

Comment:
build here, so there is one location item to fill/prefill

Reply:
👍